### ISSUE TITLE
Improvement suggestions

### ISSUE THREAD

**Issue by RobertBeilich** [cutiko/igit#1]
In reference to your article https://dev.to/cutiko/interactive-commit-using-terminal-60:

tl;dr: use aliases and auto completion

Long version:
To have an "interactive" git, just use git (as simple as it sounds).
If you just write `git commit` it will open your editor (set via the environment variable `EDITOR` (you can set sublime as this, if I'm not mistaken)) where you are able to add your commit message. This further allows you to not only have the one-line commit message you are getting with `git commit -am` (by doing that you can add other valuable information. Refer to [this](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) superb article by Tim Pope on how to write good commit messages)

On your topic on not wanting to write it all the time:
Assuming you are using bash just add `alias gc='git commit'` to your .bashrc (or `alias gcam='git commit -am'`) You can use whatever alias you want, of course.

For `git add` and `git remove`, you could better use [this](https://github.com/git/git/blob/master/contrib/completion/git-completion.bash) completion file (just add it to your `bin` folder (which you added to your `PATH`) and use the tab key for completion.
If you don't know what it is, you want to commit, your commit probably is way to big. You should split it up into multiple commits (have a look at git status and evaluate what you want to include in your commit)

Fun fact: here in Germany we have a word called `igitt` which we use as an exclamation of disgust. Therefore the name of your repo made me smile. No offense of course! You did well in trying to make your workflow easier. It was just a funny coincidence.

**Comment by cutiko** [comment]
Greetings

Thanks for the interest in this humble repo, your considerations have been listened and will be included as a proper disclaimer in future revisions.

Prior to starting with his side project your request and others were evaluated, concluding I did feel this accommodate the intended use cases.

Thanks for participating and feel free to keep doing it.
